api/src/main/java/org/openmrs/Cohort.java

```diff
@@ -27,13 +27,14 @@
 import org.simpleframework.xml.Element;
 import org.simpleframework.xml.ElementList;
 import org.simpleframework.xml.Root;
+import org.springframework.expression.EvaluationContext;
 

```

Comment:
Actually shows up when i use the Ctrl + Shift + O option for resolving import issues